### ISSUE TITLE
[XrdSecsss] Old sss key not accepted anymore in R5

### ISSUE THREAD

**Issue by esindril** [xrootd/xrootd#1237]
We have an old sss keytab that doesn't seem to work in R5. 
What is the purpose of the following comparison (note the 0x7)? 
https://github.com/xrootd/xrootd/blob/master/src/XrdSecsss/XrdSecProtocolsss.cc#L392

For completeness the sss keytab looks like this:
```
 sudo xrdsssadmin list /etc/eos.keytab
     Number Len Date/Time Created Expires  Keyname User & Group
     ------ --- --------- ------- -------- -------
          2  32 03/14/11 22:47:20 -------- eos-test daemon daemon
```

**Comment by abh3** [comment]
That's a very good question. The key is padded to be a multiple of eight bytes (line 687). So, the check wants to make sure that has happened. I see that statement 687 is incorrect when the keysize is exactly 8 characters long, sigh. It's a bug.

**Comment by abh3** [comment]
Oh, yes this only affects R5 clients. Fix coming your way.

**Comment by abh3** [comment]
Fixed with commit e6a7b0b
